### PR TITLE
Update README and include in docs

### PR DESCRIPTION
**Pull request by banesullivan**
To address https://github.com/girder/large_image/issues/692#issuecomment-980505571

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,7 +18,21 @@ Large Image |build-status| |codecov-io| |license-badge| |doi-badge|
...
    :target: https://zenodo.org/badge/latestdoi/45569214
 
-Python modules to work with large multiresolution images.
+.. |pypi-badge| image:: https://img.shields.io/pypi/v/large-image.svg?logo=python&logoColor=white
+   :target: https://pypi.org/project/large-image/
+
+*Python modules to work with large, multiresolution images.*
+
+Large Image is developed and maintained by the Data & Analytics group at Kitware, Inc. for processing large geospatial and medical images. This provides the backbone for several of our image analysis platforms including `Resonant GeoData <https://github.com/ResonantGeoData/ResonantGeoData>`_ and TODO add some others.
+
```
Comment (banesullivan) [inline README.rst:27]: @manthey, can you add another you think is relevant

File: README.rst [hunk 2]
```diff
@@ -16,7 +18,21 @@ Large Image |build-status| |codecov-io| |license-badge| |doi-badge|
...
+
+Highlights
+----------
+
+- Tile serving made easy
+- Supports a wide variety of geospatial and medical image formats
+- Convert to tiled Cloud Optimized (Geo)Tiffs
+
```
Comment (banesullivan) [inline README.rst:35]: Any addition highlights you want to convey, @manthey?

**Review by manthey — APPROVED** [review]
I like these changes.  Should we move the section about upgrading from Girder 2 to another document?  It is old enough that it probably doesn't have much utility in the main readme.

**Comment by banesullivan** [comment]
I think it would make sense to move that somewhere in the sphinx docs and outside of the readme

**Comment by manthey** [comment]
> I think it would make sense to move that somewhere in the sphinx docs and outside of the readme

Let's do that in a separate PR.

**Pull request merged**